Exercise 4/creationalPattern/BuilderPattern.java

```diff
@@ -0,0 +1,49 @@
+package creationalPattern;
+
+class SmartPhone{
+	String processor,camera,storage,ram;
+	
+	public SmartPhone(BuildPhone components){
+		this.processor = components.processor;
+		this.camera = components.camera;
+		this.storage = components.storage;
+		this.ram = components.ram;
+	}
+
+	static class BuildPhone{
```

Comment:
Check whether static is needed here and remove if not needed

Reply:
The static is needed so that I could build a phone without creating an object for inner class